libs/ui-watt/.storybook/main.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   core: { builder: 'webpack5' },
 
   stories: [
+    '../src/lib/intro.stories.mdx',
```

Comment:
Well, in this case `.mdx` seem to work. Not sure why it doesn't with a component.